core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -15,6 +15,13 @@
 
     <div ng-repeat="question in data.questionList.getQuestions() track by $index">
       <anchor-header header-id="question.getId()" header-text="Question <[$index +1]>"></anchor-header>
+      <anchor-header header-id="getSubfieldId(question,'Multiple Choice')"
+                     header-text="<[getQuestionLabel(question.getInteractionId())]>" ng-click="chooseQuestionType = true;">
```

Comment:
Pull ng-click's value into its own function.

Maybe call it: questionTypeIsBeingChosen; the current name sounds more like a verb rather than something you assign a value to.